### PR TITLE
Explicit API for contract precompilation.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

**Review by bowenwang1996 — COMMENTED** [review]
Does this change the protocol?

**Comment by olonho** [comment]
> Does this change the protocol?

Nope, just adds the API.

**Review by ailisp — APPROVED** [review]

File: runtime/near-vm-runner/src/runner.rs [hunk 1]
```diff
@@ -144,6 +145,35 @@ pub fn run_vm_profiled<'a>(
...
+) -> Option<VMError> {
+    use crate::cache::compile_and_serialize_wasmer;
+    match vm_kind {
+        VMKind::Wasmer => {
+            let result = compile_and_serialize_wasmer(code, wasm_config, code_hash, cache);
+            if result.is_err() {
+                result.err()
+            } else {
+                None
+            }
```
Comment (evgenykuzyakov) [inline runtime/near-vm-runner/src/runner.rs:168]:
```suggestion
            result.err()
```

**Comment by olonho** [comment]
PTAL

**Review by evgenykuzyakov — APPROVED** [review]

**Pull request merged**